appengine/mail/handle_catchall.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
confused why we need this file then? and also the app.yaml refers to handle_catchall which doesn't seem to exist